lib/mixlib/install/backend/base.rb

```diff
@@ -175,6 +175,25 @@ def normalize_platform(platform, platform_version)
 
           [platform, platform_version]
         end
+
+        #
+        # Normalizes architecture information that we receive.
+        #
+        # @param [String] architecture
+        #
+        # @return String [architecture]
+        def normalize_architecture(architecture)
+          case architecture
```

Comment:
replaced the if/else logic. It was there for a `fuzzy` search method I added to the Array class. We don't need that anymore.